### PR TITLE
chore(deps): support Cypress 12

### PR DESCRIPTION
**Pull request by piwysocki**

### CODE REVIEW

**Comment by piwysocki** [comment]
"realMouseMove default option.position is 'topLeft'" test fails

File: cypress/e2e/mouse.cy.ts [hunk 1]
```diff
@@ -401,29 +401,29 @@ describe("canvas drag with realMouseMove", () => {
...
      * but all other siblings should have unique `points` attributes,
      * so they should be clickable.
      */
-      cy.get("body")
+    cy.get("body")
       .realMouseDown()
       .realMouseMove(20, 10)
       .realMouseMove(30, 20)
       .realMouseUp();
-
```
Comment (dmtrKovalenko) [inline cypress/e2e/mouse.cy.ts:419]: Can you please undo any formatting changes?

Comment (piwysocki) [reply cypress/e2e/mouse.cy.ts]: very unwillingly

**Review by dmtrKovalenko — APPROVED** [review]

**Pull request merged**